client/src/components/ReviewTab/ReviewTab.tsx

```diff
@@ -38,7 +72,36 @@ export default function PersonalInfo(): JSX.Element {
             <ReviewTabForm handleSubmit={handleSubmit} />
           </Grid>
         </Grid>
-      </Grid>
+      </Paper>
+      <Typography align="center" component="h1" variant="subtitle1" className={classes.readAll}>
```

Comment:
Please reserve the usage of header tags for main titles/headings (for accessibility reasons)

Reply:
sure :)